README.md

```
@@ -1,6 +1,23 @@
- My Digital Portfolio site
+# Stephen Wanjala -  Portfolio
+> A showcase of my projects and skills.
+
+### Prerequisites
```

markdownlint-cli2 (0.21.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)